docs/pages/about.tsx

```diff
@@ -0,0 +1,5 @@
+import { NextPage } from 'next';
+
+const about: NextPage = () => <div>About</div>;
```

Comment:
```suggestion
const About: NextPage = () => <div>About</div>;
```

We capitalize React components.